test/evergreen.yml

```diff
@@ -770,6 +788,8 @@ tasks:
       - name: compile-ubsan
     commands:
       - func: "fetch artifacts"
+        vars:
+          dependent_task: compile-ubsan
```

Comment:
This is to fix a bug that was found during my testing with the `ubsan-test` task.